src/java.desktop/share/classes/sun/awt/AppContext.java

```diff
@@ -397,6 +397,13 @@ public static boolean isMainContext(AppContext ctx) {
      */
     @SuppressWarnings({"deprecation", "removal"})
     public void dispose() throws IllegalThreadStateException {
+        System.err.println(
+            """
+            WARNING: sun.awt.AppContext.dispose() no longer stops threads.
+            Additionally AppContext will be removed in a future release.
+            Remove all uses of this internal class as soon as possible.
+            There is no replacement.
+            """);
```

Comment:
Should some portion of this warning be included in an `@implNote` in this method's JavaDoc?

Reply:
Since it is internal API I think that unnecessarily generous.
It shouldn't be called anyway.

Reply:
Sounds reasonable.

Reply:
Maybe in time it can be removed, which removes the temptation to use it completely.